### PR TITLE
Add angle65

### PR DESCRIPTION
**Pull request by Vertex-kb**
## Description
Add angle65

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/vertex/angle65/config.h [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+/* key matrix size */
+#define MATRIX_ROWS 5
+#define MATRIX_COLS 15
+
+#define MATRIX_COL_PINS { B9, B8, B7, B6, B5, B4, B3, B11, A15, A10, A9, B14, B13, B12, A5 }
+#define MATRIX_ROW_PINS { B10, B1, B0, A7, A6 }
+#define DIODE_DIRECTION COL2ROW
+
+#define LED_CAPS_LOCK_PIN C13
+#define LED_PIN_ON_STATE 0
```
Comment (waffle87) [inline keyboards/vertex/angle65/config.h:29]:
```suggestion
#define MATRIX_COL_PINS { B9, B8, B7, B6, B5, B4, B3, B11, A15, A10, A9, B14, B13, B12, A5 }
#define MATRIX_ROW_PINS { B10, B1, B0, A7, A6 }
```

File: keyboards/vertex/angle65/config.h [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+/* disable debug print */
+//#define NO_DEBUG
+
+/* disable print */
+//#define NO_PRINT
+
+/* disable action features */
+//#define NO_ACTION_LAYER
+//#define NO_ACTION_TAPPING
+//#define NO_ACTION_ONESHOT
```
Comment (waffle87) [inline keyboards/vertex/angle65/config.h:68]:
```suggestion
#define FORCE_NKRO
```

File: keyboards/vertex/angle65/info.json [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    "manufacturer": "vertex",
+    "url": "",
+    "maintainer": "EasonQian1, Vertex-kb",
+    "usb": {
+        "vid": "0x7374",
+        "pid": "0x9770",
+        "device_version": "0.0.1"
+    },
+    "processor": "STM32F103",
+    "bootloader": "stm32duino",
```
Comment (waffle87) [inline keyboards/vertex/angle65/info.json:12]:
```suggestion
    "bootloader": "stm32duino",
    "diode_direction": "COL2ROW",
    "indicators": {
        "caps_lock": "C13",
        "on_state": 0
    },
```

File: keyboards/vertex/angle65/rules.mk [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no      # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no       # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+SLEEP_LED_ENABLE = no
+
+HAPTIC_ENABLE = yes
+HAPTIC_DRIVER += SOLENOID
+
+DEFAULT_FOLDER = vertex/angle65
```
Comment (waffle87) [inline keyboards/vertex/angle65/rules.mk:18]:
```suggestion
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

File: keyboards/vertex/angle65/config.h [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+#define MATRIX_COL_PINS { B9, B8, B7, B6, B5, B4, B3, B11, A15, A10, A9, B14, B13, B12, A5 }
+#define MATRIX_ROW_PINS { B10, B1, B0, A7, A6 }
```
Comment (drashna) [inline keyboards/vertex/angle65/config.h:19]: These should be moved into the info.json

Comment (Vertex-kb) [reply keyboards/vertex/angle65/config.h]: Corrected, thank you

**Review by drashna — APPROVED** [review]

File: keyboards/vertex/angle65/rules.mk [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no      # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no       # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+SLEEP_LED_ENABLE = no
```
Comment (fauxpark) [inline keyboards/vertex/angle65/rules.mk:13]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by waffle87 — APPROVED** [review]

**Pull request merged**